lua/telescope/pickers/layout_strategies.lua

```diff
@@ -563,6 +597,12 @@ layout_strategies.vertical = make_documented_layout("vertical", vim.tbl_extend("
     prompt.line = results.line + results.height + 2
   end
 
+  if tbln then
+    prompt.line = prompt.line + 1
+    results.line = results.line + 1
+    preview.line = preview.line ~= nil and preview.line + 1
```

Comment:
Hmm why is preview.line only not a thing for vertical. It seems like all other configs set it even if previewer is disabled. (Just asking stupid questions, you know way more about layout)

Reply:
Not a stupid question, it was bugging me too. Just couldn't figure out a way of refactoring it that didn't seem arbitrary.

Latest commit fixes this (and also allows the `mirror` option to work without a previewer, which will be very useful when we manage to get previewer toggle working).